library/src/main/java/com/otaliastudios/zoom/ZoomEngine.kt

```diff
@@ -1128,6 +1131,7 @@ internal constructor(context: Context) : ViewTreeObserver.OnGlobalLayoutListener
      */
     override fun moveTo(@Zoom zoom: Float, @AbsolutePan x: Float, @AbsolutePan y: Float, animate: Boolean) {
         if (!mInitialized) return
+        cancelFling()
```

Comment:
Not needed in the `animate` case, as `setState()` there will cancel what's needed.

Reply:
Correct, I only did it to make sure it is always stopped... but I guess it doesn't add much to double cancel it.